client/src/styles/history.scss

```
@@ -58,10 +58,7 @@ div.elemental-area__element--historic.elemental-area__element--historic-inner {
 
 .elemental-area--read-only {
   border-bottom: 1px solid $border-color-light;
-  margin-bottom: ($panel-padding-y * 1.5);
-  margin-top: ($panel-padding-y * 1.5);
-  margin-left: -$panel-padding-x;
-  margin-right: -$panel-padding-x;
+  margin: ($panel-padding-y * 1.5) - $panel-padding-x;
```

```suggestion
  margin: ($panel-padding-y * 1.5) -$panel-padding-x;
```
I don't that `- 5px` works, while `-5px` does

---

Oops yup, for some reason I was thinking this is a math expression, but that's not how spaces work without calc. Will fix.

---

Done